ci/report-tests.ps1

```diff
@@ -66,6 +60,12 @@ Catch {
 $test_results_url = "https://buildkite.com/organizations/$env:BUILDKITE_ORGANIZATION_SLUG/pipelines/$env:BUILDKITE_PIPELINE_SLUG/builds/$env:BUILDKITE_BUILD_ID/jobs/$env:BUILDKITE_JOB_ID/artifacts/$test_results_id"
 $test_log_url = "https://buildkite.com/organizations/$env:BUILDKITE_ORGANIZATION_SLUG/pipelines/$env:BUILDKITE_PIPELINE_SLUG/builds/$env:BUILDKITE_BUILD_ID/jobs/$env:BUILDKITE_JOB_ID/artifacts/$test_log_id"
 
+# Read the test results
```

Comment:
Why this?

Reply:
So, the results of this parsing are only needed below, when building the slack attachment, so they can be moved. But the uploading of the artifacts happens *after* parsing the test results, which means that if parsing test results fails (e.g. if the editor crashed while running tests, meaning there is no `index.json` to parse) - the uploading of the logs as artifacts would never be run, so you wouldn't get any debugging information for why your editor crashed in CI.